docs/proposals/release-notes.md

```diff
@@ -115,14 +115,15 @@ hundreds of entries.  The goal is to highlight the major changes for a release.
 
 The munger/bot option fits most cleanly into the existing workflow.
 
-The design will include:
+All accounting is managed on the master branch PR only.
+No `release-note-*` labels needed on cherry-pick PRs and no information
+will be collected from that cherry-pick PR.
 
 1. New labels added to github: `release-note-none`, maybe others for new release note categories - see Layout section below
 1. A [new munger](https://github.com/kubernetes/kubernetes/issues/23409) that will:
-  * Initiate a `release-note-needed` label on all new PRs
+  * Initiate a `release-note-needed` label on all new master branch PRs
```

Comment:
Yes, thanks for catching.  That was one of the many things that had a few versions.